moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -1,35 +1,31 @@
-  <arg name="gazebo_gui" default="true"/>
-  <arg name="initial_joint_positions" doc="Initial joint configuration of the robot"
-       default="[GAZEBO_INITIAL_JOINTS]"/>
+  <!-- Gazebo specific options -->
+  <arg name="world_name" doc="Gazebo world file" default="worlds/empty.world"/>
+  <arg name="world_pose" doc="Robot spawn pose argument" default="-x 0 -y 0 -z 0 -R 0 -P 0 -Y 0"/>
+  <arg name="initial_joint_positions" doc="Initial joint configuration of the robot argument" default="[GAZEBO_INITIAL_JOINTS]"/>
 
-  <!-- startup simulated world -->
-  <include file="$(find gazebo_ros)/launch/empty_world.launch">
-    <arg name="world_name" default="worlds/empty.world"/>
+  <!-- Start Gazebo paused -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch" pass_all_args="true">
+    <arg name="world_name" value="$(arg world_name)"/>
     <arg name="paused" value="true"/>
-    <arg name="gui" value="$(arg gazebo_gui)"/>
   </include>
 
-  <!-- send robot urdf to param server -->
-  <param name="robot_description" [GAZEBO_URDF_LOAD_ATTRIBUTE] />
+  <!-- Set robot urdf on the parameter server -->
+  <param name="robot_description" [GAZEBO_URDF_LOAD_ATTRIBUTE]/>
 
-  <!-- unpause only after loading robot model -->
-  <arg name="unpause" value="$(eval '' if arg('paused') else '-unpause')" />
-  <!-- push robot_description to factory and spawn robot in gazebo at the origin, change x,y,z arguments to spawn in a different position -->
-  <arg name="world_pose" value="-x 0 -y 0 -z 0" />
-  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model robot $(arg unpause) $(arg world_pose) $(arg initial_joint_positions)"
-    respawn="false" output="screen" />
+  <!-- Push robot_description to factory, spawn robot in Gazebo and unpause -->
+  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model robot -unpause $(arg world_pose) $(arg initial_joint_positions)" output="screen"/>
 
   <!-- Load joint controller parameters for Gazebo -->
-  <rosparam file="$(find [GENERATED_PACKAGE_NAME])/config/gazebo_controllers.yaml" />
+  <rosparam file="$(find [GENERATED_PACKAGE_NAME])/config/gazebo_controllers.yaml"/>
+
   <!-- Spawn Gazebo ROS controllers -->
-  <node name="gazebo_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="joint_state_controller" />
+  <node name="gazebo_controller_spawner" pkg="controller_manager" type="spawner" output="screen" args="joint_state_controller"/>
+
   <!-- Load ROS controllers -->
   <include file="$(dirname)/ros_controllers.launch"/>
 
   <!-- Given the published joint states, publish tf for the robot links -->
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true" output="screen" />
-
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true" output="screen"/>
```

Comment:
You have a few distracting white-space-only changes here. Please consider reverting them.